packages/grafana-runtime/src/utils/sessionStorage.ts

```diff
@@ -0,0 +1,11 @@
+export const getReturnToPrevious = () => {
+  return {
+    title: window.sessionStorage['returnToPreviousTitle'],
+    href: window.sessionStorage['returnToPreviousHref'],
+  };
+};
+
+export const setReturnToPrevious = ({ title, href }: { title: string; href: string }) => {
+  window.sessionStorage['returnToPreviousTitle'] = title;
+  window.sessionStorage['returnToPreviousHref'] = href;
```

Comment:
Does this actually work? I thought `sessionStorage` is read-only and `getItem` or `setItem` methods should be used with it.

Reply:
Yes, it works 🤷 But you're right, in the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage#examples) the examples use `sessionStorage.setItem('key', 'value')` so I'll change it now.
Thanks for the shout! 🤗